loaders/httpfs/loader.go

```diff
@@ -0,0 +1,30 @@
+package httpfs
+
+import (
+	"io"
+	"net/http"
+)
+
+type httpFileSystemLoader struct {
+	http.FileSystem
+}
+
+func NewLoader(fs http.FileSystem) httpFileSystemLoader {
```

Comment:
I will make a new package jettest that we can later use to move the httpfs tests to his own package. let's return the original type for now then.

So, i will merge you commits now, again thanks for the pull request.